geospaas_harvesting/crawlers.py

```diff
@@ -99,17 +99,18 @@ class WebDirectoryCrawler(Crawler):
         f'^.*/{YEAR_PATTERN}/{MONTH_PATTERN}/{DAY_OF_MONTH_PATTERN}/.*$')
     DAY_OF_YEAR_MATCHER = re.compile(f'^.*/{YEAR_PATTERN}/{DAY_OF_YEAR_PATTERN}(/.*)?$')
 
-    def __init__(self, root_url, time_range=(None, None), excludes=None):
+
+    def __init__(self, root_url, time_range=(None, None), includes=None):
```

Comment:
Since you use `self.include` and `self.EXCLUDE` (singular) in the rest of the code, maybe name this parameter `include`?

Reply:
DOne in the next commit